nodes-validators/networks/mainnet/become-a-full-node-operator-from-source.md

```diff
@@ -13,7 +13,7 @@ This tutorial assumes that your node is running Ubuntu LTS version (i.e: 18.04,
 ### Go version (required)
 
 The Golang version should be from 1.18 and above
-If you have not installed it yet, you can refer to [this document](https://github.com/oraichain/docs/blob/master/developer/tutorials/install-go.md).
+If you have not installed it yet, you can refer to [this document]((https://github.com/iammortimer/docs/blob/master/nodes-validators/tutorials/install-go.md)).
```

Comment:
Do you mind changing the url to this location instead: https://github.com/oraichain/docs/blob/master/nodes-validators/tutorials/install-go.md

Reply:
Apologies, copied the wrong link.